src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetInstanceWithProperties.sql

```diff
@@ -0,0 +1,48 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetInstanceV6
+--
+-- FIRST SCHEMA VERSION
+--     6
+--
+-- DESCRIPTION
+--     Gets valid dicom instances at study/series/instance level with additional instance properties
+--
+-- PARAMETERS
+--     @invalidStatus
+--         * Filter criteria to search only valid instances
+--     @partitionKey
+--         * The Partition key
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetInstanceWithProperties (
```

Comment:
I think we could have also re-used the name `GetInstance` and including the properties at the end is incidental. But I'm also ok with this more explicit name

Reply:
This is to extend it future. The idea is to have a light weight get just the Ids and get it with additional properties.